Makefile

```diff
@@ -81,51 +81,14 @@ examples_clean:
 python_library_install:
 	@echo "\n::\033[34m Installing Razer python library\033[0m"
 	@echo "====================================================="
-	@install -v -d gui/lib/razer $(DESTDIR)/$(PYTHONDIR)/razer
-	@cp -v -r gui/lib/razer/* $(DESTDIR)/$(PYTHONDIR)/razer
+	@install -v -d pylib $(DESTDIR)/$(PYTHONDIR)/razer
+	@cp -v -r pylib/* $(DESTDIR)/$(PYTHONDIR)/razer
 
 python_library_uninstall:
 	@echo "\n::\033[34m Uninstalling Razer python library\033[0m"
 	@echo "====================================================="
 	rm -rf $(DESTDIR)/$(PYTHONDIR)/razer
 
-# Python Tray App
-python_tray_app_install:
-	@echo "\n::\033[34m Installing Razer python tray applet\033[0m"
-	@echo "====================================================="
-	@install -v -d gui/tray_applet $(DESTDIR)/usr/share/razer_tray_applet
-	@cp -v -r gui/tray_applet/* $(DESTDIR)/usr/share/razer_tray_applet
-
-python_tray_app_uninstall:
-	@echo "\n::\033[34m Uninstalling Razer python tray applet\033[0m"
-	@echo "====================================================="
-	rm -rf $(DESTDIR)/usr/share/razer_tray_applet
-
-# Python Chroma App
-python_chroma_app_install:
-	@echo "\n::\033[34m Installing Razer python chroma applet\033[0m"
-	@echo "====================================================="
-	@install -v -d gui/chroma_controller $(DESTDIR)/usr/share/razer_chroma_controller
-	@cp -v -r gui/chroma_controller/* $(DESTDIR)/usr/share/razer_chroma_controller
-
-python_chroma_app_uninstall:
-	@echo "\n::\033[34m Uninstalling Razer python chroma applet\033[0m"
-	@echo "====================================================="
-	rm -rf $(DESTDIR)/usr/share/razer_chroma_controller
-
-# App desktop files
-desktop_files_install:
```

Comment:
Removing this target causes the make file to fail when building with either `make ubunutu_install` (https://github.com/pez2001/razer_chroma_drivers/blob/f856e05dc0531d026ca5d8a26e62141c7df1d4eb/Makefile#L149) or `make install` (https://github.com/pez2001/razer_chroma_drivers/blob/f856e05dc0531d026ca5d8a26e62141c7df1d4eb/Makefile#L197).